android/build.gradle

```diff
@@ -51,7 +51,7 @@ repositories {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
-    implementation 'io.sentry:sentry-android:4.3.0'
+    implementation 'io.sentry:sentry-android:5.0.0'
```

Comment:
similar to https://github.com/getsentry/sentry-react-native/pull/1560#discussion_r646348368